sklearn/neighbors/_classification.py

```diff
@@ -144,6 +131,19 @@ class KNeighborsClassifier(KNeighborsMixin, ClassifierMixin, NeighborsBase):
        training data.
 
     https://en.wikipedia.org/wiki/K-nearest_neighbor_algorithm
+
+    Examples
+    --------
+    >>> X = [[0], [1], [2], [3]]
+    >>> y = [0, 0, 1, 1]
+    >>> from sklearn.neighbors import KNeighborsClassifier
+    >>> neigh = KNeighborsClassifier(n_neighbors=3)
+    >>> neigh.fit(X, y)
+    KNeighborsClassifier(...)
+    >>> print(neigh.predict([[1.1]]))
+    [0]
+    >>> print(neigh.predict_proba([[0.9]]))
+    [[0.66666667 0.33333333]]
```

Comment:
To avoid floating point issues with different platforms:

```suggestion
    [[0.666... 0.333...]]
```

Reply:
Thanks for reviewing my PR! Should I include this suggestion to the current PR?